features/step_definitions/custom_web_steps.rb

```diff
@@ -243,6 +245,10 @@ def take_screenshots_with_login
 end
 
 And /^I click close on all the popovers$/ do
+  page.execute_script("$('.popover .close')[0].click();")
+  page.should have_selector(".popover", count: 2)
+  page.execute_script("$('.popover .close')[0].click();")
+  page.should have_selector(".popover", count: 1)
```

Comment:
All right. I don't know the difference att all TBH. I only used it to delay the closing between the popover to solve a timing issue.